news_aggregator/core/browser_pool.py

```
@@ -106,17 +124,141 @@ def get_browser_pool_status() -> BrowserPoolStatus:
+        return False
     _last_failure_at = None
     _last_failure_error = None
+    _consecutive_browser_failures = 0
+    _restart_marker_latched = False
+    return True
+
+
+def _is_transport_failure(error: BaseException) -> bool:
+    if isinstance(
+        error,
+        (ConnectionError, OSError, EOFError) + _WEBSOCKET_TRANSPORT_ERRORS,
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)